classes/task/FileLoader.inc.php

```diff
@@ -126,7 +127,39 @@ function executeActions() {
 		if (!$this->checkFolderStructure()) return false;
 
 		$foundErrors = false;
+		// get setting value
+		$memoryLimit = ini_get('memory_limit');
+		// multiply for shorthand notation
+		switch (strtolower($memoryLimit[strlen($memoryLimit)-1])) {
```

Comment:
True.  It works because PHP interprets "[0-9]+[a-z]" in numeric context as just the number.  Could have been clearer, but I think we're dropping this whole approach in favor of https://github.com/pkp/pkp-lib/pull/354 .